docs/walkthrough.md

```diff
@@ -8,9 +8,6 @@ If you haven't, please see the [installation instructions](./install.md). Option
 the Service Catalog CLI, svcat. Examples for both svcat and kubectl are provided
 so that you may follow this walkthrough using svcat or using only kubectl.
 
-All commands in this document assume that you're operating out of the root
-of this repository.
```

Comment:
but basically, k8s is doing that on their examples: 
```
kubectl apply -f https://k8s.io/examples/controllers/nginx-deployment.yaml
```
but just using that with `k8s.io` domain.
_source: https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#creating-a-deployment_

In the future, we can also host examples under https://svc-cat.io site.

thanks to that you can copy-paste command directly to terminal without service catalog sources, so it's huge pros. What's more, you will know that the newest version was used.
In case of doing that with checkout repo, someone can have an old version (didnt execute `git pull` for a long time)

but as you said such an approach requires the connection connectivity 

So what is your idea? Leave it as it is and required that use checkout the service catalog locally as it was before?

Reply:
ok, I will implement that in next pull-requests